quantized_mesh_encoder/encode.py

```diff
@@ -51,13 +51,14 @@ def encode(f, positions, indices, bounds=None, sphere_method=None):
           - None: Runs both the naive and the ritter methods, then returns the
             smaller of the two. Since this runs both algorithms, it takes around
             500 µs on my computer
+        - ellisoid: a dict that defines an ellipsoid with with "a" and "b" values. Defaults to WGS84
     """
 
     # Convert to ndarray
     positions = positions.reshape(-1, 3).astype(np.float32)
     indices = indices.reshape(-1, 3).astype(np.uint32)
 
-    header = compute_header(positions, sphere_method)
+    header = compute_header(positions, sphere_method, ellipsoid)
```

Comment:
Nit: after the first two args I usually try to use keyword args
```suggestion
    header = compute_header(positions, sphere_method, ellipsoid=ellipsoid)
```